components/merkledb/src/proto/mod.rs

```diff
@@ -0,0 +1,68 @@
+// Copyright 2019 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Module of the rust-protobuf generated files.
+
+// For protobuf generated files.
+#![allow(bare_trait_objects)]
+
+pub use self::list_proof::*;
+use exonum_crypto::proto::*;
+
+include!(concat!(env!("OUT_DIR"), "/protobuf_mod.rs"));
+
+#[cfg(test)]
```

Comment:
Could make sense to cover errors (e.g., `ProofListKey` components being out of bounds). Could be done separately, I suppose.

Reply:
Sure, will do in the following PR.